scripts/generate.sh

```diff
@@ -0,0 +1,14 @@
+echo "Regenerating account client"
```

Comment:
```suggestion
#!/bin/bash
echo "Regenerating account client"
```

Linters can give more useful information if the target shell is specified.